src/Servers/HttpSys/src/FeatureContext.cs

```diff
@@ -619,10 +619,10 @@ private async Task NotifiyOnStartingAsync()
                 return;
             }
 
-            actions.Reverse();
             // Execute last to first. This mimics a stack unwind.
-            foreach (var actionPair in actions)
+            for (var i = actions.Count; i++ > 0; )
```

Comment:
```suggestion
            for (var i = actions.Count - 1; i >= 0; i--)
```

Reply:
If you care that much about it, I have no problem.

Reply:
The code as written doesn't work does it? It's failing a bunch of tests:
https://dev.azure.com/dnceng/public/_build/results?buildId=908050&view=ms.vss-test-web.build-test-results-tab&runId=28956888&resultId=123998&paneView=debug

Reply:
OOPS! Typo!